bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/tools/messages.properties

```diff
@@ -0,0 +1,113 @@
+#Eclipse modern messages class
+#Wed May 18 13:39:42 CEST 2022
+SizeDescriptionText=Size\: 
+RemoveToolMessageBox=Do you want to delete the selected tools?
+RemoveToolMessageBoxFinish=Do you want to continue or finish the changes?
+SystemPathMessage=System Path\: 
+ToolsTreeSizeCol=Size (MB)
+FilterTargets=Filter Chips
+UrlDescriptionText=URL\: 
+ToolsTreeNameCol=Name
+Available=Available
+WarningMessagebox=IDF_PATH not defined please select the IDF Directory from the next dialog
+InstallPreRquisitePage=Install Prerequisites
+ManageToolsInstallationDescription=Here you can install tools available with the selected IDF version
+InstallEspIdfPage_lblEspidfPath_text=ESP-IDF Path\:
+ManageToolsInstallation=Tools Installation
+InstallToolsText=Install Tools
+InstallEspIdfPage_lblEspidfVersion_text=ESP-IDF Version\:
+InstallEspIdfPage_btnDownload_text=Download
+Warning=Warning
+ExtractionTextMessage=Extracting file\: 
+PreviousToolMessage=Previous Tool Path\: 
+DeselectAllButton=Deselect All
+DeselectAllButtonToolTip=Deselect all tools
+GitLabel=Git\: 
+DirectorySelectionDialog_IDFDirLabel=ESP-IDF Directory\:
+ToolsManagerShellHeading=Manage Tools Installation
+ToolsManagerWizard=Tools Installation Wizard
+InstallEspIdfPage_Existing=Existing
+InstallEspIdfPageDescription=Download ESP-IDF or select an existing version
+ToolsTreeStatusCol=Status
+SupportedTargetsDescriptionText=Supported Targets\: 
+DownloadFileText=Downloading File\: 
+SelectRecommended=Select Recommended
+SelectRecommendedToolTip=Selects recommended tools only
+DeleteToolsText=Delete Tools
+DeleteToolsTextToolTip=Delete selected tools. Make sure not to delete the recommended tools for proper working
+FileSelectionDialogTitle=Select tool executable file
+DirectorySelectionDialog_SelectIDFDirMessage=Select ESP-IDF Directory\:
+FilterLabel=Type below to filter
+Installed=Installed
+DescriptionText=Description
+ShowAvailableVersionsOnly=Show Available Versions Only
+InstallButton=Install 
+UpdateToolPathMessage=Updated Tool Path\: 
+ManageToolsInstallationShell_mntmNewItem_text=New Item
+ExtractionCompletedMessage=Extraction Completed
+InstallEspIdfPage_btnNew_text=New
+InstallEspIdfPage=Install ESP-IDF
+ManageToolsInstallationShell_tltmCheckItem_text=Check Item
+InstallEspIdfPage_lblDownloadDirectory_text=Download Directory\:
+NotInstalled=Not Installed
+RemoveToolMessageBoxTitle=Confirmation
+UpdatingPathMessage=Updating PATH
+PythonLabel=Python\:
+InstallToolsPreReqPageDescription=Set path for the required tools here
+DownloadProgressText=Download Progress\: 
```

Comment:
i see slash is added for a lot of text messages, how does it help? Might cause an issue during translation. Please check

Reply:
fixed now